client/my-sites/site-settings/controller.js

```diff
@@ -16,6 +16,7 @@ import SiteSettingsMain from 'my-sites/site-settings/main';
 import StartOver from './start-over';
 import ThemeSetup from './theme-setup';
 import ManageConnection from './manage-connection';
+import DisconnectSite from './disconnect-site/why';
```

Comment:
Is there any reason to have the `why` here? Can't we have it all under just `disconnect-site`? That would also help make the component name make more sense.

In case there's a particular reason to have this subdirectory, would it make sense to rename it to something (a noun or a verb) that makes sense if we change the copy, like `survey` for example?